templates/single.php

```diff
@@ -11,17 +11,7 @@
 	</style>
 </head>
 <body>
-<nav class="amp-wp-title-bar">
-	<div>
-		<a href="<?php echo esc_url( $this->get( 'home_url' ) ); ?>">
-			<?php $site_icon_url = $this->get( 'site_icon_url' ); ?>
-			<?php if ( $site_icon_url ) : ?>
-				<amp-img src="<?php echo esc_url( $site_icon_url ); ?>" width="32" height="32" class="amp-wp-site-icon"></amp-img>
-			<?php endif; ?>
-			<?php echo esc_html( $this->get( 'blog_name' ) ); ?>
-		</a>
-	</div>
-</nav>
+<?php $this->load_parts( apply_filters( 'amp_post_template_meta_parts', array( 'nav' ) ) ); ?>
```

Comment:
Can we drop the filter here? It's only relevant for the `meta` section of the template.